stakingModel/contracts/InterestDistribution.sol

```diff
@@ -160,7 +162,7 @@ library InterestDistribution {
         Staker storage stakerData = _interestData.stakers[_staker];
         uint256 _withdrawnToDate = stakerData.withdrawnToDate;
 
-        uint256 intermediateInterest = stakerData.totalEffectiveStake.mul(_interestData.globalGDYieldPerToken);
+        uint256 intermediateInterest = stakerData.totalEffectiveStake.mul(_interestData.globalGDYieldPerToken).div(DECIMAL1e18);
```

Comment:
why are you dividing by 1e18 all over?
always add comments explaining non trivial calculations

Reply:
Made changes and added comments.